src/main/java/karuna/karuna_backend/email/MailjetEmailSender.java

```diff
@@ -0,0 +1,51 @@
+package karuna.karuna_backend.email;
+
+import com.mailjet.client.errors.MailjetException;
+import com.mailjet.client.MailjetClient;
+import com.mailjet.client.MailjetRequest;
+import com.mailjet.client.MailjetResponse;
+import com.mailjet.client.ClientOptions;
+import com.mailjet.client.resource.Emailv31;
+import lombok.extern.slf4j.Slf4j;
+import org.jetbrains.annotations.NotNull;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.springframework.stereotype.Service;
+
+@Service
+@Slf4j
+class MailjetEmailSender implements EmailSender {
+
+    @Override
+    public void sendEmail(String to, String subject, String contentText, String contentHtml) {
+        try {
+            MailjetClient client = buildMailjetClient();
+            MailjetRequest request = configureRequest(to, subject, contentText, contentHtml);
+            client.post(request);
+            log.info("Success sent message!");
+        } catch (MailjetException ex) {
+            log.error("Error while sending email", ex);
+        }
+    }
+
+    @NotNull
+    private static MailjetClient buildMailjetClient() {
+        return new MailjetClient(System.getenv("MJ_APIKEY_PUBLIC"), System.getenv("MJ_APIKEY_PRIVATE"));
+    }
```

Comment:
It would be beneficial to create some kind of Config as in JwtConfig. So we could get env variables from dev/prod profiles in app.properties

Reply:
Its bad idea. because we have public repos, secret api key should be got from env

Reply:
I mean ofc it should be from env variables but those env variables would be set in production profile inside pom.xml file. I believe if you use System.getenv inside code and someone does not have this env variables setup it would print an error which we want to avoid

Reply:
I make research about tests api keys, but after work 🗡️